Changes

```diff
@@ -2,8 +2,11 @@ Revision history for HTML-FormBuilder
 
```

Comment:
Then where should I put this file to ? cpan need this file. And it stores the change history.